pkg/competition/competitionservice/serv.go

```diff
@@ -21,12 +21,14 @@ type Serv interface {
 }
 
 type competitionServ struct {
-	Store *util.Store
+	Store  *util.Store
+	Config config.StaticConfig
 }
 
-func NewCompetitionServ(str *util.Store) Serv {
+func NewCompetitionServ(str *util.Store, cfg config.StaticConfig) Serv {
```

Comment:
might want to just push queue config, instead of entire static config

Reply:
Good Idea